guides/source/upgrading_ruby_on_rails.md

```diff
@@ -86,6 +86,10 @@ This means it won't be possible to load them with a manual `require` call, the c
 Reducing the size of `$LOAD_PATH` speed-up `require` calls for apps not using `bootsnap`, and reduce the
 size of the `bootsnap` cache for the others.
 
+### Active Storage's BaseController no longer includes the Streaming concern
+
+Applications that inherit from BaseController in order to create custom file serving logic must include the `ActiveStorage::Streaming` concern in those controllers. 
```

Comment:
I tweaked the wording a bit.  What do you think?

```suggestion
### `ActiveStorage::BaseController` no longer includes the streaming concern

Application controllers that inherit from `ActiveStorage::BaseController` and use streaming to implement custom file serving logic must now explicitly include the `ActiveStorage::Streaming` module.
```

Reply:
I was hoping someone would do that. I've updated the commit.